src/platform/elements/date-picker/DatePickerInput.ts

```diff
@@ -4,7 +4,7 @@ import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
 import { TAB, ENTER, ESCAPE } from '@angular/cdk/keycodes';
 // Vendor
 import { TextMaskModule } from 'angular2-text-mask';
-import createAutoCorrectedDatePipe from 'text-mask-addons/dist/createAutoCorrectedDatePipe';
```

Comment:
Don't need this anymore? Does this break what Novo is expecting?

Reply:
I confused that with another change I made earlier. I added it back in